Cargo.toml

```
@@ -1280,6 +1280,17 @@ description = "Bezier curve example showing a cube following a cubic curve"
 category = "Animation"
 wasm = true
 
+[[example]]
+name = "ease_functions"
+path = "examples/animation/ease_functions.rs"
+doc-scrape-examples = true
+
+[package.metadata.example.ease_functions]
+name = "Ease Functions"
```

Whoops missed this one in my suggestions (in case you decide to incoporate them)
```suggestion
name = "Easing Functions"
```